### PR TITLE
Setup and first tests and docs

### PR DESCRIPTION
**Pull request by profxj**
As titled

### CODE REVIEW

**Comment by profxj** [comment]
@samwang141224 -- I wished for you to review my PR not to add files to it.
Please remove all of your changes and review this PR.

You should add your changes to a separate branch and eventually submit
a separate PR for those.

Thanks!

**Comment by profxj** [comment]
Also, when you do start adding files, go somewhat slowly, 
i.e not all at once.  I'll be quick to review your PRs.  I promise.

But I am going to be more strict about documentation and tests.

**Comment by samwang141224** [comment]
sorry I misunderstood the operation,I have deleted all the adding files.

**Comment by samwang141224** [comment]
OK, since our codes include different parts:such as generating dataset, training and validation, maybe I can begin to upload codes for generating dataset?

File: desidlas/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+# Users
+numpy>=1.18.0
+scipy>=1.4
+astropy>=4.2
+IPython>=7.2.0
+pytest>=6.0.0
+pandas>=0.25
+#photutils>=1.0.1
+
```
Comment (samwang141224) [inline desidlas/requirements.txt]: maybe we can add tensorflow_gpu >2.2

**Review by samwang141224 — APPROVED** [review]

File: desidlas/requirements.txt (hunk 1 above)
Comment (profxj) [reply desidlas/requirements.txt]: done

**Pull request merged**